packages/vite/src/node/packages.ts

```diff
@@ -37,11 +36,10 @@ export interface PackageData {
   }
 }
 
-export function invalidatePackageData(
+function invalidatePackageData(
   packageCache: PackageCache,
   pkgPath: string,
 ): void {
-  packageCache.delete(pkgPath)
```

Comment:
This isn't really working now because cache keys have `rpd_` and `fnpd_` prefixes. The below `forEach` loop should cover this instead.